extras/multisigs.json

```diff
@@ -10,7 +10,9 @@
     "blabs_ops": "0x02f35dA6A02017154367Bc4d47bb6c7D06C7533B",
     "linearPoolController": "0x75a52c0e32397A3FC0c052E2CeB3479802713Cf4",
     "foundation": "0x3B8910F378034FD6E103Df958863e5c684072693",
-    "beets_treasury": "0x811912c19eEF91b9Dc3cA52fc426590cFB84FC86"
+    "beets_treasury": "0x811912c19eEF91b9Dc3cA52fc426590cFB84FC86",
+    "grants_treasury": "0xE2c91f3409Ad6d8cE3a2E2eb330790398CB23597",
+    "threeRocks_treasury": "0x3B8910F378034FD6E103Df958863e5c684072693"
```

Comment:
theeRocks is a new entity and this is a multisig with 2+ million dollars in assets in it created by @solarcurvey.  You sure this is the right safe?